dotenv.cabal

```diff
@@ -91,13 +91,13 @@ library
                        , Configuration.Dotenv.Text
                        , Configuration.Dotenv.Types
 
-  build-depends:         base >= 4.9 && < 5.0
-                       , directory
-                       , megaparsec >= 7.0.1 && < 10.0
-                       , containers
-                       , process >= 1.6.3.0 && < 1.7
-                       , shellwords >= 0.1.3.0
-                       , text
+  build-depends:         base >= 4.16.4 && < 4.17
```

Comment:
@DanRCM Updating `base` version mean that you're limiting the version of GHC that is compatible with this library. You can check the relation between GHC and base [here](https://wiki.haskell.org/Base_package). `dotenv` is tested against GHC 8.10 (base >= 4.14). Let's not change that boundary.